### PR TITLE
Log update on test execution progress

### PR DESCRIPTION
**Pull request by kedarbcs16**

### CODE REVIEW

File: framework/src/main/java/org/apache/drill/test/framework/DrillTestJdbc.java [hunk 1]
```diff
@@ -61,12 +62,14 @@
   private Statement statement = null;
   private AtomicBoolean doneProcessingResultSet = new AtomicBoolean(false);
   private int id;
-
-  public DrillTestJdbc(DrillTestCase modeler, ConnectionPool connectionPool, int id) {
+  private int totalCases;
+  private static int value;
+  public DrillTestJdbc(DrillTestCase modeler, ConnectionPool connectionPool, int id,int totalCases) {
```
Comment (Agirish) [inline framework/src/main/java/org/apache/drill/test/framework/DrillTestJdbc.java]: Add space after ',' preceding int totalCases

File: framework/src/main/java/org/apache/drill/test/framework/DrillTestJdbc.java [hunk 2]
```diff
@@ -61,12 +62,14 @@
   private Statement statement = null;
   private AtomicBoolean doneProcessingResultSet = new AtomicBoolean(false);
   private int id;
-
-  public DrillTestJdbc(DrillTestCase modeler, ConnectionPool connectionPool, int id) {
+  private int totalCases;
+  private static int value;
```
Comment (Agirish) [inline framework/src/main/java/org/apache/drill/test/framework/DrillTestJdbc.java]: A more meaningful name to the variable?

**Comment by Agirish** [comment]
Can you please rebase it on top of master? I see the regex commits here.

**Comment by Agirish** [comment]
+1 otherwise.

File: framework/src/main/java/org/apache/drill/test/framework/DrillTestJdbc.java (hunk 2 above)
Comment (kedarbcs16) [inline framework/src/main/java/org/apache/drill/test/framework/DrillTestJdbc.java]: done

**Comment by kedarbcs16** [comment]
rebased

**Pull request merged**